index.html

```diff
@@ -5,9 +5,28 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="style.css"/>
-        <title>title</title>
+        <title>Барбершоп «Бородинский»</title>
     </head>
     <body>
+        <header>
+            <nav>
+                <a>
+                <img src="svg/index-logo.svg" width="368" height="120" alt="Логотип барбершопа «Бородинский»">
+                </a>
```

Comment:
```suggestion
                <a>
                    <img src="svg/index-logo.svg" width="368" height="120" alt="Логотип барбершопа «Бородинский»">
                </a>
```
вложенность тега, таб